agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -255,9 +255,11 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
               if ("none".equals(otelConfig.getString("otel.traces.exporter"))) {
                 // in this case the spanExporter here is the noop spanExporter
                 return spanExporter;
-              } else {
+              } else if (!"otlp".equals(otelConfig.getString("otel.traces.exporter"))) {
+                startupLogger.verbose("Unsupported otel.traces.exporter: {}", otelConfig.getString("otel.traces.exporter"));
```

Comment:
I think this might be the wrong format `{}` for java.util.logging, can you check?

Reply:
i will test it to make sure that log is surfaced for non-otlp values